exercises/Loops-basic-vector-R.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: exercise
+topic: Loops
+title: Basic Vector
+language: R
+---
+
+The following code capitalizes the first letter of `pet` and stores it as
+`name`:
+
+```
+library(stringr)
+pet <- "spot"
+pet_letters <- str_split(pet, "")[[1]]
+first <- toupper(pet_letters[1])
+others <- str_c(pet_letters[-1], collapse = "")
+name <- str_c(first, others, sep = "")
+```
```

Comment:
`stringr` in general seems to be a mine field for this sort of thing (having learned this lesson the hard way several times myself. What about just using something like `cat` or `paste` instead? Making lines like, "My pet's name is spot"?